src/components/TaskBox.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+
+const TaskBox = ({ name, logoImport, link }) => {
+  const [logolink, setLogolink] = useState("");
+
+  useEffect(() => {
+    // Get logo from import
+    const importLogo = async () => {
```

Comment:
if using an async function, should await the `logoImport` instead of using `.then` (or drop the async)